src/benchmarks/real-world/ImageSharp/Images/Input/Jpg/baseline/JpegSnoopReports/Calliphora.jpg.txt

```diff
@@ -0,0 +1,331 @@
+
+JPEGsnoop 1.8.0 by Calvin Hass
```

Comment:
These files are not used by benchmarks, it' just for us to know what are the test images about.